lib/Makefile

```diff
@@ -4,9 +4,7 @@ OBJS = nx_inflate.o nx_deflate.o nx_zlib.o nx_crc.o nx_dht.o nx_dhtgen.o \
        nx_dht_builtin.o nx_adler32.o gzip_vas.o nx_compress.o nx_uncompr.o \
        crc32_ppc.o crc32_ppc_asm.o nx_utils.o sw_zlib.o
 
-VERSION ?= $(shell git describe --tags | cut -d - -f 1,2 | tr - . | cut -c 2-)
```

Comment:
With this change now the generated .so has one less `.XX`. Honestly, I'm not sure if it was actually useful to have a different version per commit, so I'm just commenting about this in case it was an unexpected change.

Before:
```
$ ll lib/libnxz.*
-rw-r--r-- 1 mscastanho users 442206 Dec  1 14:49 lib/libnxz.a
lrwxrwxrwx 1 mscastanho users     17 Dec  1 14:49 lib/libnxz.so -> libnxz.so.0.62.68
lrwxrwxrwx 1 mscastanho users     17 Dec  1 14:49 lib/libnxz.so.0 -> libnxz.so.0.62.68
-rwxr-xr-x 1 mscastanho users 298712 Dec  1 14:49 lib/libnxz.so.0.62.68
```

After:
```
$ la lib/libnxz.*
-rw-r--r-- 1 mscastanho users 442206 Dec  1 14:45 lib/libnxz.a
lrwxrwxrwx 1 mscastanho users     14 Dec  1 14:45 lib/libnxz.so -> libnxz.so.0.62
lrwxrwxrwx 1 mscastanho users     14 Dec  1 14:45 lib/libnxz.so.0 -> libnxz.so.0.62
-rwxr-xr-x 1 mscastanho users 298712 Dec  1 14:45 lib/libnxz.so.0.62
```

Reply:
It was not unintentional.
I also do not see how the third part of the file versioning was helpful.
Let's see what @rzinsly thinks about it.

Reply:
I agree with the removal, that third part don't seem that useful.